src/components/Chart.js

```diff
@@ -142,24 +142,48 @@ export default class Chart extends React.Component {
       dataTable.addColumn(column);
     });
     dataTable.addRows(this.props.rows);
+    
+    const applyNumberFormat = (column, options) => {
+      const formatter = new window.google.visualization.NumberFormat(options);
+      formatter.format(dataTable, column);
+    }
+    
+    const applyDateFormat = (column, options) => {
+      const formatter = new window.google.visualization.DateFormat(options);
+      formatter.format(dataTable, column);
+    }
+    
     if (this.props.numberFormat) {
-      const formatter = new window.google.visualization.NumberFormat(
-        this.props.numberFormat.options
-      );
-      formatter.format(dataTable, this.props.numberFormat.column);
+      const { column, options } = this.props.numberFormat;
+      this.applyNumberFormat(column, options);
```

Comment:
In line 146 it is defined as a const, and here it is `this.applyNumberFormat`.  Is it correct?